src/layers/CanvasLayer.ts

```diff
@@ -37,20 +38,12 @@ export abstract class CanvasLayer extends Layer {
     ctx.canvas.setAttribute('height', `${height}px`);
   }
 
-  onRescale(event: OnRescaleEvent): void {
-    super.onRescale(event);
-    const { xScale, yScale } = event;
-    this.onResize({ width: xScale.range()[1], height: yScale.range()[1] });
-  }
-
   onUpdate(event: OnUpdateEvent): void {
     super.onUpdate(event);
     const { ctx } = this;
     if (!ctx) {
       return;
     }
-
-    ctx.canvas.setAttribute('style', `position:absolute;z-index:${this.order};opacity:${this.opacity}`);
```

Comment:
Called onMount(). I can see that it might pose an issue if order or opacity is changed along the way, but then we should call that once when order or opacity is set and not on every update.
.

Reply:
I'll add override in relevant classes